source/patterns/@aws-solutions-constructs/core/lib/apigateway-helper.ts

```diff
@@ -191,7 +191,9 @@ export interface AddProxyMethodToApiResourceInputParams {
     readonly apiMethod: string,
     readonly apiGatewayRole: Role,
     readonly requestTemplate: string,
-    readonly contentType?: string
+    readonly contentType?: string,
+    readonly requestValidator?: api.RequestValidator,
+    readonly requestModel?: { [contentType: string]: api.IModel; }
```

Comment:
This is an odd type definition, but it matches the one from [MethodOptions.requestModels](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-apigateway.MethodOptions.html):

```typescript
  /**
   * The models which describe data structure of request payload.
   * Specify `requestModels` as key-value pairs, with a content type
   * (e.g. `'application/json'`) as the key and an API Gateway Model as the value.
   */
  readonly requestModels?: { [param: string]: IModel };
```